### DISCUSSION TITLE
Interfacial tension and contact angle in the capillary pressure curve calculation

### DISCUSSION THREAD

**Discussion by minzhangut** [PMEAL/OpenPNM#2396] [Q&A]
In this example here https://github.com/PMEAL/OpenPNM/blob/dev/examples/simulations/percolation/capillary_pressure_curves.ipynb, I am wondering whether the interfacial tension = 0.485N/m, and contact angle = 140?

**Comment by jgostick** [comment]
Yes it is.  If you look at ``hg['pore.surface_tension']`` and ``hg['pore.contact_angle']`` you can confirm this.

**Reply by jgostick** [reply]
I'm going to add one small meta comment, and I'm sorry if this sounds grumpy:  This is something you could've figured out on you own just by knowing how openpnm works on a basic level.  Please consider the value and scarceness of my time before asking a question next time.

**Reply by minzhangut** [reply]
Sorry Prof. Gostick that I have not spent enough time on openpnm… I thought I just wanted a quick confirmation from the users group since I am doing a quick comparison with the experiment data. Thanks for the message and I will be more cautious about asking questions.

**Reply by jgostick** [reply]
Don't be scared away, please ask questions if you're really stuck!  This "Discussions" forum is supposed to work like a user-group, but it seems like it's mostly for 'tech support' and it's starting to consume quite a bit of my and Amin's time.  Maybe we need to find a better way to handle this.  Anyway, good luck.

**Reply by minzhangut** [reply]
No worries, I will never be scared away. So far I have been very excited about the porespy and openpnm results. I think it is a good time for me to learn openpnm in a more systematic way.
